python/enthic/compute_stats.py

```diff
@@ -0,0 +1,234 @@
+# -*- coding: utf-8 -*-
+"""
+====================================================================
+Compute statistics from companies's data after complete and check it
+====================================================================
+
+Coding Rules:
+
+- Snake case for variables.
+- Only argument is configuration file.
+- No output or print, just log and files.
+"""
+import math
+from enthic.arbre_compte_de_resultat import TREE_VIEW
+
+def convert_data_to_tree(raw_data):
+    """
+    Returns given data as a tree like "arbre_compte_de_resultat"
+        :param raw_data : data as enthic API returns it
+        :return: given data as a tree
+    """
+    new_tree = TREE_VIEW
+    recursive_fill_tree(new_tree, raw_data)
+    return new_tree
+
+def recursive_fill_tree(tree_item, raw_data):
+    """
+    Fill the given item with data from raw_data
+        :param tree_item : item to fill, and its children
+        :param raw_data : data in Enthic API format
+    """
+
+    # Begin to fill children if there is children
+    if 'children' in tree_item:
+        for child_name in tree_item["children"]:
+            recursive_fill_tree(tree_item["children"][child_name], raw_data)
+
+    # Find corresponding value from raw_data to add to tree_item
+    i = 0
+    while i < len(raw_data):
+        for code in raw_data[i]:
+            if code in tree_item["codeLiasses"]:
+                tree_item['data'] = raw_data[i][code]
+                tree_item['data']["status"] = "official"
+                tree_item['data']['code'] = code
+                del raw_data[i]
+                break
+        i = i + 1
+
```

Comment:
PEP 8: E302 expected 2 blank lines, found 1, use autopep 8 or an IDE that reformat according to PEP8 e.g. Pycharm Ctrl+Alt+L .